backend/mlarchive/bin/scan_all.py

```diff
@@ -1046,7 +1009,7 @@ def shunt():
 def subjects(listname):
     """Return subject line of all messages for listname"""
     for msg in process([listname]):
-        print("%s: %s" % (msg.get('date'),msg.get('subject')))
+        print("%s: %s" % (msg.get('date'), msg.get('subject')))
 
 
 def subject_non_english():
```

Comment:
maybe not the best name for this function

Reply:
Addressed in commit b62adb5c2d90f16ac4713b0c6dfc3a86b0a1c0b2